core.coffee

```diff
@@ -411,6 +414,20 @@
 			delete: (params) ->
 				return @request(params, method: 'DELETE')
 
+			batch: (data) ->
+				compiledData = @compileBatch(data)
+				multipartHeader = compiledData.header('Content-Type')
+
+				params =
+					method: 'POST'
+					url: '$batch'
+					body: compiledData.toString({ noHeaders: true })
+				overrides =
+					json: false
+					headers:
+						'Content-Type': multipartHeader
+				return @request(params, overrides)
```

Comment:
The `return` is not needed right? being coffee, if you think it'll make a future migration to TS easier, then it's fine

Reply:
Personally I generally prefer having explicit returns in longer functions, but regardless it should make the typescript conversion a bit easier (and this project is pretty high on my list of needing a conversion, it's actually partly done already in the branch for next major version)